main.go

```diff
@@ -1,98 +1,148 @@
 package main
 
 import (
-	"log"
 	"os"
 
+	log "github.com/sirupsen/logrus"
+
 	"github.com/swisscom/korp/actions"
 	"github.com/urfave/cli"
 )
 
+const (
+	version = "0.0.1"
+)
+
+// main -
 func main() {
-	var scanpath, registry, output string
+
+	setLogLevel("debug")
```

Comment:
OK sure, we can do more elaborate logging, but then we should give the user an option to switch it on and off when running korp.

Reply:
Sure, we can insert it in the todo list and create another specific feature branch later :)